packages/nimble-components/src/tree-view/tests/tree-view.stories.ts

```diff
@@ -80,10 +97,50 @@ export const treeItem: StoryObj<ItemArgs> = {
         value: '1',
         disabled: false,
         icon: true,
+        selected: false,
         expanded: true
     }
 };
 
+export const anchorTreeItem: StoryObj<AnchorItemArgs> = {
+    parameters: {
+        docs: {
+            description: {
+                story: 'Use a `nimble-anchor-tree-item` to navigate to a URL from a `nimble-tree-view`. If you want a tree item that can have a value and/or child items, use a `nimble-tree-item` instead.'
```

Comment:
This is great! Can you add equivalent text to the `treeItem` story above? Now that the Docs page is more prominent in storybook I think text like this will be even more valuable.

Reply:
Done